### PR TITLE
RED-5108 - Check taskqueue ip

### PR DESCRIPTION
**Pull request by matthewbelisle-wf**
Need to check the taskqueue ip

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
<details>
<summary>(2) Vulnerable direct dependencies were detected</summary>
<li>3 vulns in <code>jinja2 < 2.11.3</code> via <code>doc/requirements.txt</code></li>
<li>3 vulns in <code>pygments < 2.7.4</code> via <code>doc/requirements.txt</code></li>
</details>
<details>
<summary>Error scanning for security relevant context</summary>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=furious">this Splunk dashboard</a> for more CVE details
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: furious/config.py [hunk 1]
```diff
@@ -71,6 +71,13 @@ def get_completion_cleanup_delay():
     return config.get('cleanupdelay')
 
 
+def get_csrf_check():
+    """Get the CSRF check function, which takes one arg (webapp2.Reqeust)
+    and returns None, throwing an exception if a CSRF attack is detected.
+    """
+    return path_to_reference(get_config().get('csrf_check'))
```
Comment (patkujawa-wf) [inline furious/config.py:78]: Do you need to `from furious.job_utils import path_to_reference` here like in `_get_configured_module` below? If not, I don't see where it's imported already.

**Review by patkujawa-wf — DISMISSED** [review]

**Review by patkujawa-wf — APPROVED** [review]

**Comment by matthewbelisle-wf** [comment]
@Workiva/release-management-p

**Comment by colefeisthamel-wf** [comment]
RM +1

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Comment by colefeisthamel-wf** [comment]
@aviary-wf

File: furious/__init__.py [hunk 2]
```diff
@@ -13,3 +13,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+import logging
+
+import webapp2
```
Comment (patkujawa-wf) [inline furious/__init__.py:19]: Doing this import at the top-level is causing some consumers' tests to have a :badtime:. I suggest moving the imports to within `csrf_check`

Comment (patkujawa-wf) [reply furious/__init__.py:19]: https://github.com/Workiva/furious/pull/189

**Pull request merged**